index.html

```diff
@@ -3484,6 +3484,31 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+
+        <p>
+Additional media types for verifiable credential representations can be
+defined to provide implementers with the ability to distinguish between representations that can be mapped to 
+the base media type (<code>application/vc+ld+json</code>).
+        </p>
+        <p>
+Specifications are registered with the [[?VC-SPECS]] and
+MUST result in a transformed, concrete serialization
+that is objectively testable via a conformance test suite.
```

Comment:
I would rather not have that requirement. 

- Does it mean we must have tests in the W3C Test Suite repo?
- Or is it ok if the conformance tests are somewhere else?
- What should those tests test, ie input file in the media type and then spitting out a JSON-LD VC?

Reply:
> What should those tests test, ie input file in the media type and then spitting out a JSON-LD VC?

Yes, that would at least point to some interop potential.
Every media type can at least "get to" the base media type.

Conceptually, the relationship between this (VCDM2.0) specification and a Media Type Representation specification (VC-JWT/Gordian/ACDC/Any) is similar to the relationship between DID-Core and a specific DID Method specification.

 By defining a Media Type section we acknowledge there will be other media types wanting to be consumed as VCs, we should be defining a minimal number of steps/spec/transform that allows someone to produce something of the base media type and know it's going to be interoperable with others.
 
 Happy to rework the language if its too constrained